### PR TITLE
Create Book class

### PR DESCRIPTION
**Pull request by aliatayee**
## create book class
In this milestone I have developed the following requirements: 
- Create a Book class in a separate .rb file.
- All Book class properties visible in the diagram should be defined and set up in the constructor method.
- can_be_archived?() in the Book class
    - should override the method from the parent class
     - should return true if parent's method returns true OR if cover_state equals to "bad"
     - otherwise, it should return false

### CODE REVIEW

**Review by bigmosi — COMMENTED** [review]
Hi, @aliatayee 
You Implemented all the features. It's time to merge.

**Review by bigmosi — APPROVED** [review]
Hi, @aliatayee 
You Implemented all the features. It's time to merge.

File: models/book.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require 'item'
+
+class Book < Item
+  attr_accessor :publisher, :cover_state, :publish_date, :archived
+
+  def initialize(publisher, cover_state, publish_date, id: nil, archived: false)
```
Comment (RitobrotoMukherjee) [inline models/book.rb:6]: **[OPTIONAL]** You can create a public instance variable `title`. Like this while listing the `Books` in the console app, you will be able to show the **name of each book**

**Review by RitobrotoMukherjee — APPROVED** [review]
Hi @aliatayee 👋 

Your project is complete. 💯 There is nothing else to say, other than it's time to merge. 🥇 

## Key Highlights
- Good PR description
- No linter errors

**[OPTIONAL]** Please check the comment I have suggested an optional change. 💪

**Review by ksigei — APPROVED** [review]
@aliatayee 
STATUS APPROVED
There is nothing else other than to merge the branch and go on the next step

**Pull request merged**